mono/metadata/boehm-gc.c

```diff
@@ -482,7 +483,7 @@ on_gc_notification (GC_EventType event)
 			mono_perfcounters->gc_gen0size = heap_size;
 		}
 #endif
-		gc_stats.major_gc_time += mono_100ns_ticks () - gc_start_time;
+		UnlockedAddUnsigned64 (&gc_stats.major_gc_time, mono_100ns_ticks () - gc_start_time);
```

Comment:
@vargaz @lewurm can I convert the `gc_stats.*_gc_time`_* values ([gc-internal-agnostic.h](https://github.com/mono/mono/pull/5381/commits/d87f71c42f3cd2ea0f4e0097952fe1f4b51585db#diff-91d6a369b7a39d55fd63be9665e86498)) to `gint64`? The reason behind this: An `InterlockedAdd64 ()` function exists already, however, I would have to "invent" `InterlockedAddUnsigned64 ()`. Furthermore, @vargaz told me at to simply use signed types if there is no explicit reason for using unsigned ones (https://github.com/mono/mono/pull/5299) and `9,223,372,036,854,775,807 ns` are still almost `300 years` ... do we really need `600 years`?